aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -682,9 +682,9 @@ std::tuple<Tensor, Tensor> _scaled_dot_product_attention_math(
     attn = at::softmax(attn, -1);
     if (dropout_p > 0.0) {
       if (dropout_mask.has_value()) {
-        auto attn_dropout_masked = attn.masked_fill(dropout_mask->logical_not(), 0.0);
+        attn = attn.masked_fill(dropout_mask->logical_not(), 0.0);
```

Comment:
should have returned the dropout filled mask which makes it more useful for debugging